app/models/comment.rb

```diff
@@ -1,4 +1,4 @@
 class Comment < ApplicationRecord
-  belongs_to :author, class_name: 'User'
-  belongs_to :post, counter_cache: :comments_counter
+  belongs_to :author, class_name: 'User', foreign_key: :author_id
+  belongs_to :post, counter_cache: :comments_counter, foreign_key: :post_id
```

Comment:
- [ ] comment should have a method that updates comments counter
```def update_comments_counter
  post.increment!(:comments_counter)
end
```

Reply:
@VICTOR-CHUKWU  Thanks for your feed back!